policy_definitions/tagging/policy_assignment_qby_deploy_require_tags.tmpl.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "QBY-Require-Tags",
+    "type": "Microsoft.Authorization/policyAssignments",
+    "apiVersion": "2022-06-01",
+    "dependsOn": [],
+    "properties": {
+      "description": "Enforce existence of conceptual defined tags.",
+      "displayName": "Require tags",
+      "enforcementMode": null,
+      "nonComplianceMessages": [
+        {
+          "message": "The resource doesn't have the required tag."
+        }
+      ],
+      "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policySetDefinitions/QBY-Require-Tags",
```

Comment:
Is there such a policyset definition? I can't see a file for that. Regardless, you should add the new require tag policies to the existing policy set.

Reply:
I think, that we don't need. Two reaseon

One, this custom policy is the same of the other two that I used before, however with not empty value.
Two, this policy is for all the resources, and in QBY-Deploy-Tag-Structure, we have that a tags in resources group will inherits to all the resources of the resources group.

So, I think, that we only need, the definition and the assgnment.

However, if I am in a error. Please, tell me.

Reply:
Right now you new policy definition is not used, as there is no assignment. This assignement refers a non existing policySet `QBY-Require-Tags`. To use a policy definition, there are two scenarios: 

1. Policy Definition assigned
2. Policy definition referenced in a policy set definition and this policy set definition is assigned

As far as I can tell tell, there is no PolicySetDefinition names `QBY-Require-Tags`, only a Policy Definition named that way.